src/components/about-chat-sidebar/AboutChatSidebar.styles.ts

```diff
@@ -0,0 +1,57 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '24px',
+    p: '24px 8px',
+    maxWidth: '320px',
+    overflowY: 'scroll'
+  },
+  header: {
+    text: {
+      color: 'primary.700'
+    },
+    iconButton: {
```

Comment:
you can just create to properties, for example:
headerText: { color: 'primary.700'}
headerIcon: { color: 'primary.800, p: '8px' }

Reply:
> you can just create two properties, for example: headerText: { color: 'primary.700'} headerIcon: { color: 'primary.800, p: '8px' }

fixed